.github/bin/github-pages-setup.sh

```diff
@@ -79,7 +79,7 @@ wget https://raw.githubusercontent.com/IQAndreas/markdown-licenses/master/apache
 
 # Generate lint rules documentation
 # --------------------------------
-bazel-bin/verilog/tools/lint/verible-verilog-lint -generate_markdown > $PAGES_DIR/lint.md
+verible-verilog-lint -generate_markdown > $PAGES_DIR/lint.md
```

Comment:
We should not rely on the binaries being in the path, but rather we look for it in a directory we specify (makes everything easier to understand).

I suggest to have an optional parameter to `github-pages-setup.sh [<bindir>]` which then is used to assemble the binary path:

```sh
#!/usr/bin/env bash
BINDIR=${1:-/usr/bin}
# ...
${BINDIR}/verible-verilog-lint ...
```

Reply:
Oh, I like this one a lot. I was actually having a conundrum how to solve this in an elegant way and this suggestion seems to be it.

Reply:
Done.